duxiu/test/duxiu.2019-04-16.csv

```diff
@@ -0,0 +1,41 @@
+320150885025;4E5400CB12C210379C0120228B11B8BD;;REF;HTML;http://book.duxiu.com:80/StdDetail.jsp?dxid=320150885025&d=4E5400CB12C210379C0120228B11B8BD
+390102513141;C90969D30D1392AC1689F64A75418331;;REF;HTML;http://jour.duxiu.com:80/thesisDetail.jsp?dxNumber=390102513141&d=C90969D30D1392AC1689F64A75418331
+400112204339;A5C339C6E04F539F99907F7C6EB28114;;REF;HTML;http://book.duxi.com:80/godownexam.jsp?dxid=400112204339&d=A5C339C6E04F539F99907F7C6EB28114
+400100671692;63CDE68AB1857D663BDBD25D7BA230A9;;REF;HTML;http://book.duxiu.com:80/courseDetail.jsp?dxid=400100671692&d=63CDE68AB1857D663BDBD25D7BA230A9&sw=mind&ecode=utf-8
+360118685645;9259B2820DFD44F2BF758DA1042B19ED;;REF;HTML;http://book.duxiu.com:80/InfoDetail.jsp?dxNumber=360118685645&d=9259B2820DFD44F2BF758DA1042B19ED
+403710174382;381ABC9CE704439D88399C30752F2025;;REF;HTML;http://book.duxiu.com:80/EncyDetail.jsp?dxid=403710174382&d=381ABC9CE704439D88399C30752F2025
+403713385167;4DB9B4BC318ED0D763FF1E5D8C97FE0B;;REF;HTML;http://book.duxiu.com:80/EncyDetail.jsp?dxid=403713385167&d=4DB9B4BC318ED0D763FF1E5D8C97FE0B&stitle=carrot
+400505987423;4823920D633F84E1A5831BF35AC1CDB9;;REF;HTML;http://vrefer.duxiu.com:80/publicinfoDetail?dxid=400505987423&d=4823920D633F84E1A5831BF35AC1CDB9
+400504254654;2AD1CD09C6341D721FA9D521F5BD5AA8;;REF;HTML;http://vrefer.duxiu.com:80/publicinfoDetail?dxid=400504254654&d=2AD1CD09C6341D721FA9D521F5BD5AA8
+000016213781;BF1D22D4A90B8ADF9CF61B0CCCB1676C;;BOOK PART;HTML;http://book.duxiu.com:80/readDetail.jsp?dxNumber=000016213781&d=BF1D22D4A90B8ADF9CF61B0CCCB1676C&t=1
+000005229290;D2D23AD5C1D5E69A0680AEC4A4E9AA2E;;BOOK PART;HTML;http://book.duxiu.com:80/readDetail.jsp?dxNumber=000005229290&d=D2D23AD5C1D5E69A0680AEC4A4E9AA2E&t=3
+mind;;;REF;HTML;http://book.duxiu.com:80/searchDict?Field=all&channel=searchDict&sw=mind&ecode=utf-8&edtype=&searchtype=&view=0
+mind+of+convict+tattoo;;;REF;HTML;http://book.duxiu.com:80/searchDict?sw=mind+of+convict+tattoo&channel=searchDict&ecode=utf-8
+000030080757;188F458266335F85D0B1C7D2B4EEB69D;;BOOK PART;PDF;http://qw.duxiu.com:80/godownpdf.jsp?kid=65676B686B686C6D3637353530333734&pagenum=99&epage=101&a=188F458266335F85D0B1C7D2B4EEB69D&bt=qw&dxNumber=000030080757&sch=brain&zjid=000030080757_37
+000001309122;8743606FBD35AAB644E9C6EE8FB93B0F;;BOOK PART;PDF;http://qw.duxiu.com:80/godownpdf.jsp?kid=6969696D6A6E6B6C3238313035373338&pagenum=109&epage=109&a=8743606FBD35AAB644E9C6EE8FB93B0F&bt=qw&dxNumber=000001309122&sch=brain&zjid=000001309122_132
+000016139576;477110DE29765CD58BBA62F46000971A;;BOOK PART;PDF;http://qw.duxiu.com:80/godownpdf.jsp?kid=63666265636B696A3638333139383732&pagenum=29&epage=32&a=477110DE29765CD58BBA62F46000971A&bt=qw&dxNumber=000016139576&sch=Brain-BasedTeaching%A3%A6Learning&zjid=000016139576_29
+400271577283;097B8C533DF3A071F202C9E497E77C29;;PDF;PDF;http://book.duxiu.com:80/godowndoc.jsp?dxid=400271577283&d=097B8C533DF3A071F202C9E497E77C29
```

Comment:
lines 38 through 41 please change the out-rtype from PDF to whatever it should be ;)